### ISSUE TITLE
Tenant spoofing

### ISSUE THREAD

**Issue by gr4per** [OpusCapita/auth#113]
We need to cover the support Use Case:

# Support user can spoof a tenant in order to inspect data / help solve application issues

Support user gets a call for application issue by authorized client.
Support user determines tenant used by that client and then chooses "Switch Tenant" option from user menu which is visible for admin users.
Switch tenant allows user to select type (customer/supplier) of tenant and select tenant by typing directly and using autocomplete. 
Once user is assigned to tenant a new id token is issued to reflect the same.
Support user also gets supplier-admin / customer-admin role for the spoofed tenant while assigned and loses the admin role while assigned.

The user can logout / re -login in order to return to privileged non-tenant-associated admin account.

# Implementation

Basically this is supplier reference search (or customer reference search). And could be reusing shared components for both if we have a separate control allowing tenant type selection.

**Comment by sureshvarman-cronj** [comment]
This feature is ready to integrate with react menu.
